homeassistant/data_entry_flow.py

```diff
@@ -120,7 +121,7 @@ async def async_init(
             raise UnknownFlow("Flow was not created")
         flow.hass = self.hass
         flow.handler = handler
-        flow.flow_id = uuid.uuid4().hex
```

Comment:
Let's keep these as uuid4 as these are used for the login flow.

Reply:
will revert 👍